Gravity/Gravity.Test.Unit/MockedRsapiProviderTestBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+
+namespace Gravity.DAL.RSAPI.Tests
+{
+	public class MockedRsapiProviderTestBase
+	{
+		protected Mock<IRsapiProvider> rsapiProvider;
+
+		[SetUp]
+		public void Init()
+		{
+			rsapiProvider = new Mock<IRsapiProvider>(MockBehavior.Strict);
+		}
+
+		[TearDown]
+		public void End()
+		{
+			if (!true.Equals(TestContext.CurrentContext.Test.Properties["SkipVerifyAll"].FirstOrDefault()))
```

Comment:
I was a bit confused reading this statement at first. Maybe it could be simplified a bit?